tests/foreman/api/test_smartproxy.py

```diff
@@ -6,11 +6,11 @@
 
 :CaseLevel: Acceptance
```

Comment:
@lpramuk It would be great if we change Caselevel as component here. I don't see any instance of acceptance caselevel in this file.

Reply:
or yet better remove ```:CaseLevel:``` from module level and have it only defined for every single test

Reply:
Ok. Updated ```:CaseLevel:```